Wrappers/Python/cil/processors/RingRemover.py

```diff
@@ -174,5 +176,7 @@ def xRemoveStripesVertical(self,ima, decNum, wname, sigma):
         for i in range(decNum-1,-1,-1):
             nima = nima[0:Ch[i].shape[0],0:Ch[i].shape[1]]
             nima = pywt.idwt2((nima,(Ch[i],Cv[i],Cd[i])),wname)
-            
-        return nima      
\ No newline at end of file
+        
+        nima_crop = nima[original_extent[0], original_extent[1]]
```

Comment:
You can return the cropped range directly rather than assigning it to a variable.

Reply:
Updated this